src/index.js

```diff
@@ -29,6 +29,10 @@ const _init = (options) => {
     return;
   }
 
+  if (options.iFrameOption) {
```

Comment:
@patrick-fs If we wanted to not update the cog, the window variable could be set here.